README.md

```diff
@@ -318,7 +318,7 @@ pub fn verify_work(start: BaseElement, result: BaseElement, proof: StarkProof) {
     // The number of steps and options are encoded in the proof itself, so we
     // don't need to pass them explicitly to the verifier.
     let pub_inputs = PublicInputs { start, result };
-    match winterfell::verify::<WorkAir>(proof, pub_inputs) {
+    match winterfell::verify::<WorkAir, WorkProver::Hash>(proof, pub_inputs) {
```

Comment:
Instead of referring to `WorkProver` here, I'd prefer using the hash function directly. Or, if that's too long, maybe create a type alias for it.